src/ONE.Models.CSharp/Enterprise/Twin/Wizard/Location.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace ONE.Models.CSharp.Enterprise.Twin.Wizard
+{
+    public class Location
+    {
+        // Is this a default in the wizard
+        public bool d { get; set; }
```

Comment:
These names are not good, is this data being deserialized from something that has equally poor naming?

Reply:
renamed.